src/main/java/org/jfrog/bamboo/config/EncryptionHelper.java

```diff
@@ -0,0 +1,160 @@
+package org.jfrog.bamboo.config;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.bouncycastle.util.encoders.Base32;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import javax.annotation.concurrent.ThreadSafe;
+import javax.crypto.*;
+import javax.crypto.spec.DESedeKeySpec;
+import java.nio.charset.StandardCharsets;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import java.security.spec.InvalidKeySpecException;
+import java.util.Base64;
+
+/**
+ * Created by diman on 07/03/2017.
+ */
+@ThreadSafe
+public class EncryptionHelper {
+
+    private static final Logger log = LogManager.getLogger(EncryptionHelper.class);
+
+    private static final String DESEDE_ENCRYPTION_SCHEME = "DESede";
```

Comment:
Let's use AES, since DES is no longer considered safe
```suggestion
    private static final String DESEDE_ENCRYPTION_SCHEME = "AES";
```